### ISSUE TITLE
plugins prefixIds or cleanupIDs don't prefix?

### ISSUE THREAD

**Issue by v3nt** [svg/svgo#985]
Heya - really struggling with svg IDs and these plugins.

I can't seem to affect the IDs whatever i try - i have tried a lot. The svgs do get processed but the cleanIds and prefixIds plugins below don't seem to affect the output in any way.

All of the plugins are in the plugins folder.

removeViewBox seems to be the only thing i can activate and deactivate. Ideally i want to change all ids to data-id so i can still style / animate all my svgs. 

```
gulp.task('images', function() {
    return gulp.src(globs.images).pipe(imagemin([
        imagemin.jpegtran({
            progressive: true
        }),
        imagemin.gifsicle({
            interlaced: true
        }),
        imagemin.svgo({
            plugins: [
            {
                removeViewBox:true //works
            },
           {
                cleanupIDs: {
                    remove: false,
                    minify: false,
                    prefix: 'xxx',
                    force: true
                }
            }, {
                prefixIds: true
            }]
        })
    ])).pipe(gulp.dest(path.dist + 'images')).pipe(browserSync.stream());
});
```

```
npm  -v
6.1.0
npm list gulp-svgo
gulp-svgo@1.5.4
```

Is this a bug or something i'm doing wrong? As this is such a common issue feels like it should be easier to solve!

**Comment by lidoma** [comment]
@v3nt I have the same issue. any update?

**Comment by strarsis** [comment]
@v3nt, @lidoma: Are you using a recent enough `svgo`?

**Comment by TrySound** [comment]
Probably fixed in 2.0.1
